extensions/cornerstone-dicom-sr/src/viewports/OHIFCornerstoneSRViewport.tsx

```diff
@@ -0,0 +1,450 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import OHIF, { utils } from '@ohif/core';
+import DICOMSRDisplayTool from './../tools/DICOMSRDisplayTool';
+import SRLengthTool from '../tools/annotationTools/SRLengthTool';
+import { setTrackingUniqueIdentifiersForElement } from '../tools/modules/dicomSRModule';
+import {
+  Notification,
+  ViewportActionBar,
+  useViewportGrid,
+  useViewportDialog,
+} from '@ohif/ui';
+
+const { formatDate } = utils;
+
+const MEASUREMENT_TRACKING_EXTENSION_ID =
+  '@ohif/extension-measurement-tracking';
+
+const SR_TOOLGROUP_BASE_NAME = 'SRToolGroup';
+
+function OHIFCornerstoneSRViewport(props) {
+  const {
+    children,
+    dataSource,
+    displaySets,
+    viewportIndex,
+    servicesManager,
+    extensionManager,
+  } = props;
+
+  const { DisplaySetService, ToolGroupService } = servicesManager.services;
+
+  // SR viewport will always have a single display set
+  const srDisplaySet = displaySets[0];
```

Comment:
Instead, can we do a check that will throw if displaySets has a length > 1? That seems a lot cleaner.